packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -162,6 +168,7 @@ class DropdownMenu<T> extends StatefulWidget {
     this.focusNode,
     this.requestFocusOnTap,
     this.expandedInsets,
+    this.filterCallback,
     this.searchCallback,
     required this.dropdownMenuEntries,
     this.inputFormatters,
```

Comment:
I wonder if there are any assertions we can make to enforce the parameters. Like should we assert that when filterCallback is non-null, then enableFilter must be true?